mne/viz/tests/test_raw.py

```diff
@@ -279,15 +279,13 @@ def test_plot_raw_selection(raw, new_mpl):
     _click_ch_name_helper(fig, ch_index=1, button=1)  # mark good
     assert sel_fig.lasso.ec[:, 0].sum() == 0          # all channels black
     # test lasso
-    sel_fig._set_custom_selection()  # lasso empty → should do nothing
     sensor_ax = sel_fig.mne.sensor_ax
     # Lasso with 1 mag/grad sensor unit (upper left)
     _fake_click(sel_fig, sensor_ax, (0, 1), xform='ax')
     _fake_click(sel_fig, sensor_ax, (0.65, 1), xform='ax', kind='motion')
     _fake_click(sel_fig, sensor_ax, (0.65, 0.7), xform='ax', kind='motion')
     _fake_click(sel_fig, sensor_ax, (0, 0.7), xform='ax', kind='release')
-    want = ['MEG 0121', 'MEG 0122', 'MEG 0123'] if new_mpl else \
-        ['MEG 0111', 'MEG 0112', 'MEG 0131', 'MEG 0132', 'MEG 0133']  # XXX old
+    want = ['MEG 0121', 'MEG 0122', 'MEG 0123']
     want = sorted(want)
```

Comment:
Looks like this should be reverted

https://travis-ci.org/github/mne-tools/mne-python/jobs/741413151#L6453-L6460